subworkflows/local/input_check.nf

```diff
@@ -9,33 +9,43 @@ workflow INPUT_CHECK {
     samplesheet // file: /path/to/samplesheet.csv
 
     main:
-    SAMPLESHEET_CHECK ( samplesheet )
+    // Process the samplesheet and get the initial channels
```

Comment:
Is this the normal way or would it make more sense to change the python script samplesheet_check module to add an additional number?

Reply:
The parsing of the samplesheet will be changed with the new template anyways to use nf-validation. So I would rather see how to find a solution during the template merge :)